src/Service.Tests/SqlTests/RestApiTests/Patch/MsSqlPatchApiTests.cs

```diff
@@ -190,6 +208,79 @@ public async Task PatchOneViewBadRequestTest()
             await base.PatchOneViewBadRequestTest(expectedErrorMessage);
         }
 
+        /// <summary>
+        /// Test to validate that even when an update DML trigger is enabled on a table, we still return the
+        /// latest data as it is present after the trigger gets executed. Whenever an update DML trigger is enabled,
+        /// we use a subsequent SELECT query to get the data instead of using OUTPUT clause. To validate that the data is returned
+        /// as it, is after the trigger is executed, we use the values which are updated by the trigger in the WHERE predicates of the verifying sql query.
```

Comment:
```suggestion
        /// after the trigger is executed, we use the new values (written by the trigger) in the WHERE predicates of the verifying sql query.
```
phrasing awkward, i think this captures intent?